app/dockerdwrapper.c

```diff
@@ -80,6 +80,8 @@ static const char *tls_certs[] = {"ca.pem",
                                   "server-cert.pem",
                                   "server-key.pem"};
 
+#define CERT_FILE_MODE  0444
```

Comment:
not sure it needs to be readable by anyone else than the acap-user

Reply:
Done